docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -441,6 +441,18 @@ Learn how to configure the native projects in the [installation instructions in
 
 No usage description is required, see [notification-related permissions](#fetch-information-about-notifications-related-permissions).
 
+### Interpret the iOS permissions response
+
+On iOS, permissions for sending notifications are a little more granular than they are on Android.
+Because of this, you should rely on the `NotificationPermissionsStatus`'s `ios.status` field, instead of the root `status` field.
+This value will be one of the following, accessible under `Notifications.IosAuthorizationStatus`:
```

Comment:
```suggestion
On iOS, permissions for sending notifications are a little more granular than they are on Android. This is why you should rely on the `NotificationPermissionsStatus`'s `ios.status` field, instead of the root `status` field.

This value will be one of the following, accessible under `Notifications.IosAuthorizationStatus`:
```

Let's also format it correctly for our own maintenance. Without a new line between, all the lines render as a single paragraph.